packages/seedless-onboarding-controller/src/index.ts

```diff
@@ -12,6 +12,8 @@ export type {
   SeedlessOnboardingControllerStateChangeEvent,
   SeedlessOnboardingControllerActions,
   SeedlessOnboardingControllerEvents,
+  SeedlessOnboardingControllerAllowedEvents,
```

Comment:
Can you explain the purpose of exporting these types? We don't typically need to do this in controller packages. `SeedlessOnboardingControllerActions` and `SeedlessOnboardingControllerEvents` are already available, and the other action and event types that `SeedlessOnboardingMessenger` uses are already available from other packages.

Reply:
ping @mcmire 

@lwin-kyaw Btw, there is a contributor docs entry for this.
https://github.com/MetaMask/core/blob/main/docs/controller-guidelines.md#define-but-do-not-export-a-type-union-for-external-action-types
(I don't have a strong opinion on this, just noting.)

Reply:
Btw2, the upcoming messenger migration will make some of this obsolete.
See https://github.com/MetaMask/core/pull/6364.
 https://github.com/MetaMask/core/blob/15d3a73ea75fec24f472debc5cae0a0cb9ef8685/packages/seedless-onboarding-controller/tests/__fixtures__/mockMessenger.ts#L18-L21

Reply:
Btw3, you can also extract these types manually.
https://github.com/MetaMask/core/blob/42df019122b90ee813110a09b6e11c8c9d589e4b/packages/shield-controller/tests/mocks/messenger.ts#L21-L22